libs/remix-ui/renderer/src/lib/renderer.tsx

```diff
@@ -56,11 +50,14 @@ export const Renderer = ({ message, opt = {}, plugin }: RendererProps) => {
     setClose(false)
   }, [message, opt])
 
-  const getPositionDetails = (msg: any) => {
+  const getPositionDetails = (msg: string) => {
     const result = { } as Record<string, number | string>
 
     // To handle some compiler warning without location like SPDX license warning etc
-    if (!msg.includes(':')) return { errLine: -1, errCol: -1, errFile: msg }
+    if (!msg.includes(':')) return { errLine: -1, errCol: -1, errFile: '' }
```

Comment:
Please check if a case if possible where msg includes `-->`, but not `:`. In that case, it will be returned from here itself